db/migrate/20230503102449_schema.rb

```diff
@@ -0,0 +1,4 @@
+class Schema < ActiveRecord::Migration[7.0]
+  def change
+  end
+end
```

Comment:
- [ ] You are supposed to create a migration file for creating user table, one for creating posts table, one for creating comments table and one for creating likes table but I can't see it in your project. Please just add that to your project.